src/main/java/org/sonarlint/intellij/ui/SonarLintRulePanel.kt

```diff
@@ -297,12 +292,39 @@ class SonarLintRulePanel(private val project: Project, private val parent: Dispo
 
             }
             SwingHelper.setHtml(securityHotspotHeaderMessage, htmlStringBuilder.toString(), JBUI.CurrentTheme.ContextHelp.FOREGROUND)
-            headerPanel.update(project, serverFindingKey, finding.status, finding.file, ruleDescription.key, ruleDescription.type, finding.vulnerabilityProbability)
+            headerPanel.update(
+                project,
+                serverFindingKey,
+                finding.status,
+                finding.file,
+                ruleDescription.key,
+                ruleDescription.type,
+                finding.vulnerabilityProbability
+            )
         } else {
             headerPanel.update(ruleDescription.key, ruleDescription.type, ruleDescription.severity)
         }
     }
 
+    private fun securityHotspotDetailsLink(
+        serverConnection: ServerConnection,
+        projectKey: String,
+        serverFindingKey: String,
+    ): String {
+        val prefixPath = if (serverConnection.isSonarCloud) "project/" else ""
+        return "${serverConnection.hostUrl}/{$prefixPath}security_hotspots?id=${urlEncode(projectKey)}&hotspots=${
+            urlEncode(
+                serverFindingKey
+            )
+        }"
+    }
+
+    private fun securityHotspotsDocLink(serverConnection: ServerConnection) = externalLink(
+        "Security Hotspot",
+        if (serverConnection.isSonarCloud) "https://docs.sonarcloud.io/digging-deeper/security-hotspots/"
+        else "https://docs.sonarqube.org/latest/user-guide/security-hotspots/"
```

Comment:
Random idea but would it make sense to centralize all these static URLs into a specific class? In case we might want to review them in the future to see which one or deprecated or not

Reply:
good point. I will treat it separately